core/amber/src/main/scala/Engine/Common/tuple/Advanced/CompactTuple.scala

```diff
@@ -1,9 +1,9 @@
-package Engine.Common.AmberTuple.Advanced
+package Engine.Common.tuple.Advanced
```

Comment:
In package names, what is the standard regarding first letter? I think we should do either all small or all capital. Right now it's a mixed case,

Reply:
it should be all small

Reply:
We can do a global renaming in a next PR. Another thing to change is to add package name "edu.uci.ics.texera" to the new engine. Currently it doesn't have a package name.